From cb4216a709e9364e3c17ddebd7cdfb9a85089391 Mon Sep 17 00:00:00 2001
From: Gabriel Rauter <rauter.gabriel@gmail.com>
Date: Sat, 22 Aug 2020 10:53:58 +0200
Subject: [PATCH] Fix application icon name in baobab-main-window.ui

Use the correct named icon for the window. This is needed to get the
right icon on mingw builds.
---
 data/ui/baobab-main-window.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/ui/baobab-main-window.ui b/data/ui/baobab-main-window.ui
index b677cb5..2590abc 100644
--- a/data/ui/baobab-main-window.ui
+++ b/data/ui/baobab-main-window.ui
@@ -64,7 +64,7 @@
   </object>
   <template class="BaobabWindow" parent="HdyApplicationWindow">
     <property name="title" translatable="yes">Disk Usage Analyzer</property>
-    <property name="icon_name">baobab</property>
+    <property name="icon_name">org.gnome.baobab</property>
     <child>
       <object class="GtkBox" id="vbox">
       <property name="orientation">vertical</property>
-- 
2.31.1

